ir/instr.cpp

```diff
@@ -1331,6 +1333,12 @@ StateValue FnCall::toSMT(State &s) const {
   auto ret = s.addFnCall(fnName_mangled.str(), move(inputs), move(ptr_inputs),
                          out_types, !(flags & NoRead), !(flags & NoWrite),
                          flags & ArgMemOnly, move(returned_val));
+
+  if (flags & NoReturn) {
+    // TODO: Even if a function call doesn't have noreturn, it can possibly
```

Comment:
can you open an issue in Alive2 with a test case that we miss miscompilation pls?